apps/demo/src/app/cart/components/cart/cart.component.scss

```diff
@@ -1,10 +1,43 @@
 @import 'utilities';
 @import 'theme';
 
-.demo-cart {
-	display: block;
+:host {
+	display: grid;
+	grid-template-columns: 1fr;
 
-	&__content {
+	@include breakpoint(big-tablet) {
+		grid-template-columns: 2fr 1fr;
+		column-gap: 50px;
+	}
```

Comment:
All of these other styles need to be under a `.demo-cart` or they won't apply. So like:
```
:host {
   //host styles
}

.demo-cart {
  &__content {
  }

  etc
}
```